lfs/batcher.go

```diff
@@ -8,21 +8,41 @@ package lfs
 //   * Exit() is called
 // When a timeout, Flush(), or Exit() occurs, the group may be smaller than the
 // batch size.
+
+// A Lot represents a group of Transferables that was packaged up by a Batcher.
+type Lot []Transferable
```

Comment:
I don't love the name 'Lot' I had to look up the definition (and it makes sense!) but it's not a commonly used word in this way, especially not in programming. Is there  a reason we don't use 'Batch" or "GroupOfTransferables" or something like that that's a bit more clear?